filament/src/RenderPass.h

```diff
@@ -253,6 +253,7 @@ class RenderPass {
         uint16_t instanceCount;                                         // 2 bytes [MSb: user]
         Variant materialVariant;                                        // 1 byte
         uint8_t rfu1;                                                   // 1 byte
+        uint32_t rfu2;                                                  // 4 byte
```

Comment:
Great, I also found this while correcting the debug issue. Could you correct the comment below within the Command struct regarding the size of this struct?

Reply:
done